lib/lndclient/LndClient.ts

```diff
@@ -817,6 +828,25 @@ class LndClient extends SwapClient {
     this.invoiceSubscriptions.set(rHash, invoiceSubscription);
   }
 
+  /**
+   * Subscribes to channel backups
+   */
+  public subscribeChannelBackups = () => {
+    if (!this.lightning) {
+      throw errors.UNAVAILABLE(this.currency, this.status);
+    }
+
+    if (this.channelBackupSubscription) {
+      this.channelBackupSubscription.cancel();
```

Comment:
Does it make sense just to return if a channel subscription already exists, rather than cancel it and restart it? Just not sure if there's any value to doing the latter.